### ISSUE TITLE
contact losing precision ?

### ISSUE THREAD

**Issue by bmcclenahan** [wit-ai/wit#1002]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
bug
**What is the current behavior?**
the contact entity seems to have lost a bit of precision, including non-names in the entity.
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
one repeatable behaviour is to ask "What's Matt's phone number" - it returns What's Matt's as the contact entity
**What is the expected behavior?**
Just return names
**What is the App ID where you are experiencing this issue (if applicable)?**

**Comment by patapizza** [comment]
Hi @bmcclenahan,

Thanks for reporting. Do you have more examples?

**Comment by bmcclenahan** [comment]
No, that's the main one, other than that the contact entity is working quite well.

I spent some time trying to train it out but didn't get anywhere, so I thought you'd like to know as it was a fairly obvious fail.

Thanks for asking
Bruce
________________________________
